src/formats/images/jpg.js

```
@@ -0,0 +1,837 @@
+        case 0xFFE6:
+        case 0xFFE7:
+        case 0xFFE8:
+        case 0xFFE9:
+        case 0xFFEA:
+        case 0xFFEB:
+        case 0xFFEC:
+        case 0xFFED:
+        case 0xFFEE:
+        case 0xFFEF:
+        case 0xFFFE:
+          var appData = readDataBlock();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)